walker_test.go

```diff
@@ -35,6 +35,21 @@ import (
 	fspb "github.com/google/fswalker/proto/fswalker"
 )
 
+type testWriter struct {
+	outpath string
+}
+
+func (t *testWriter) writeWalk(walk *fspb.Walk) error {
+	if t.outpath == "" {
```

Comment:
Maybe this can be removed to reduce logic in tests? I don't see `outpath` being empty used anywhere in the tests unless I missed it.

Reply:
Yeah not in the tests indeed, thanks